.travis.yml

```diff
@@ -70,10 +70,14 @@ matrix:
           ./cc-test-reporter upload-coverage;
         fi
   exclude:
-    - rvm: 2.6.0
+    - rvm: 2.6.2
       gemfile: gemfiles/rails_4_2.gemfile
   fast_finish: true
 
+before_install:
+  - "[[ $BUNDLE_GEMFILE == *rails_4_2* ]] && gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true"
```

Comment:
why this change?

Reply:
because [bundler 2 does not play well with rails 4.2](https://travis-ci.org/lynndylanhurley/devise_token_auth/jobs/506927792)